share/nuopc_shr_methods.F90

```diff
@@ -760,6 +762,169 @@ end subroutine timeInit
 
 !===============================================================================
 
+  integer function get_minimum_timestep(gcomp, rc)
```

Comment:
Why is this function here - I thought it was no longer needed. Does anyone call it?

Reply:
It is called by the cmeps driver as well as by all of the cdeps component models.

Reply:
Okay - thanks for the clarification.